data-semantics.md

```diff
@@ -0,0 +1,117 @@
+Secondarily, an optional **component** tag can be provided to scope the operation name. Typically, a process, framework, library, or module name is a good guideline for the component tag.
+
+### Span Structure
+
+Span structure is also important: what do spans represent, and what relationship do they have towards their parent span?  This is covered in the [Semantic Specification](/spec).
+
+
+## Span Tag Use-Cases
+
+The following tags are recommended for instrumentors who are trying to represent a particular type of data. Tag names follow a general structure of namespacing; the values are determined purely on a tag-by-tag basis.
+
+The recommended tags below are accompanied by `const` values included in an `ext` module for each opentracing implementation.  These `ext` values should be used in place of the strings below, as tracers may choose to use different underlying representations for these common concepts.  The symbols are similar in each implementation: ([Go](https://github.com/opentracing/opentracing-go/blob/master/ext/tags.go), [Python](https://github.com/opentracing/opentracing-python/blob/master/opentracing/ext/tags.py), etc.)
+
+Some tags mentioned below may contain values of significant size. Handling of such values is implementation-specific: it is the responsibility of the Tracer to honor, drop, or truncate these tags as appropriate. However, care should be exercised on the part of the instrumentor, as even the generation or passing of such values to the Tracer may create undesirable overhead for the application.
+
+It is not required that all suggested tags be used, even if one is used.
+
+### Component Identification
+
+For any span, it can be useful to specify the type of component being instrumented. This is particularly recommended for instrumentation provided for libraries or modules, where end-users may haev a mix of custom and library-provided instrumentation.
+
+* `component` - string
+    - Low-cardinality identifier of the module, library, or package that is instrumented.
+    - Examples:
+        - `httplib` for instrumentation of Python builtin httplib functionality
+        - `JDBC` for instrumentation of JDBC database connectors
+        - `mongoose` for instrumentation of Ruby MongoDB client connector
+
+
+### HTTP Server Tags
+
+These tags are recommended for spans marking entry into a HTTP-based service.
+
+* `http.url` - string
+    - URL of the request being handled in this segment of the trace, in [standard URI format](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier), excepting query string parameters.
+    - Protocol optional
+    - Examples:
+        - `https://domain.net/path/to?resource=here`
+        - `domain.net/path/to/resource`
+        - `http://user:pass@domain.org:8888`
+* `http.query` - string
+    - Query string parameters of the request being handled.
+    - Note that the preceding `?` character is not expected.
+    - Examples:
+        - `key=val&key2=val2`
+* `http.method` - string
```

Comment:
I like the idea of `http.method` as a tag, but how does this fit in with the guidance about HTTP verbs as operation names? (C.f. my comment that they might not be the best example of operation nameS)